### PR TITLE
Add release notes 2.10.11

### PR DESCRIPTION
**Pull request by serinamarie**
[Rendered](https://github.com/PrefectHQ/prefect/blob/03a8a766c34bc9b32c2dfb4524953633dc9e991e/RELEASE-NOTES.md)
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix inconsistent flow run parameters when creating a new flow run or editing a deployment — https://github.com/PrefectHQ/prefect/pull/9705
+- Fix max value when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
+- Allow table artifacts `table` argument as list of lists — https://github.com/PrefectHQ/prefect/pull/9732
+- Use `PrefectHttpxClient` in cloud communication — https://github.com/PrefectHQ/prefect/pull/9724
+- Remove trailing slashes when building events/in websocket url — https://github.com/PrefectHQ/prefect/pull/9663
```
Comment (pleek91) [inline RELEASE-NOTES.md:28]:
```suggestion
- Remove trailing slashes when building events/in websocket url — https://github.com/PrefectHQ/prefect/pull/9663
- Update flow run logs empty text for light mode
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
+- Emit events from deployments, work queues, and work pools. — https://github.com/PrefectHQ/prefect/pull/9635
 - Improve SQLite database transaction behavior — https://github.com/PrefectHQ/prefect/pull/9594
+- Add support for SQLAlchemy 2 — https://github.com/PrefectHQ/prefect/pull/9656
+- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
 - Add an `/api/ready check` endpoint to Prefect server — https://github.com/PrefectHQ/prefect/pull/9701
 - Guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
 - Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
 - Handle deleted deployments gracefully — https://github.com/PrefectHQ/prefect/pull/9464
 - Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
```
Comment (madkinsz) [inline RELEASE-NOTES.md:19]: "telemetry" is a bit of a loaded term — can we talk about the kind of worker events we are emitting now?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,60 @@
...
+- Emit events from deployments, work queues, and work pools. — https://github.com/PrefectHQ/prefect/pull/9635
+- Improve SQLite database transaction behavior — https://github.com/PrefectHQ/prefect/pull/9594
+- Add support for SQLAlchemy 2 — https://github.com/PrefectHQ/prefect/pull/9656
+- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
+- Add an `/api/ready check` endpoint to Prefect server — https://github.com/PrefectHQ/prefect/pull/9701
+- Guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Handle deleted deployments gracefully — https://github.com/PrefectHQ/prefect/pull/9464
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
```
Comment (pleek91) [inline RELEASE-NOTES.md:19]:
```suggestion
- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
- Add "Retrying" to ui states select — https://github.com/PrefectHQ/prefect-ui-library/pull/1410
- Update docs links in ui to concepts when possible — https://github.com/PrefectHQ/prefect-ui-library/pull/1351
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
 - Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
-- Add support for SQLAlchemy 2 — https://github.com/PrefectHQ/prefect/pull/9656
-- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
 
 ### Enhancements
+- Emit events from deployments, work queues, and work pools. — https://github.com/PrefectHQ/prefect/pull/9635
 - Improve SQLite database transaction behavior — https://github.com/PrefectHQ/prefect/pull/9594
+- Add support for SQLAlchemy 2 — https://github.com/PrefectHQ/prefect/pull/9656
+- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
 - Add an `/api/ready check` endpoint to Prefect server — https://github.com/PrefectHQ/prefect/pull/9701
```
Comment (madkinsz) [inline RELEASE-NOTES.md:14]:
```suggestion
- Add an `/api/ready` endpoint to the Prefect server to check database connectivity — https://github.com/PrefectHQ/prefect/pull/9701
```

**Review by pleek91 — COMMENTED** [review]
Just adding some stuff from the ui library

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,60 @@
 # Prefect Release Notes
 
+## Release preview
```
Comment (madkinsz) [inline RELEASE-NOTES.md:3]:
```suggestion
## Release 2.10.11
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,60 @@
...
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix inconsistent flow run parameters when creating a new flow run or editing a deployment — https://github.com/PrefectHQ/prefect/pull/9705
+- Fix max value when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
+- Allow table artifacts `table` argument as list of lists — https://github.com/PrefectHQ/prefect/pull/9732
+- Use `PrefectHttpxClient` in cloud communication — https://github.com/PrefectHQ/prefect/pull/9724
+- Remove trailing slashes when building events/in websocket url — https://github.com/PrefectHQ/prefect/pull/9663
+
+### Deprecations
+- Rename `docker` modules to avoid conflict — https://github.com/PrefectHQ/prefect/pull/8788
```
Comment (madkinsz) [inline RELEASE-NOTES.md:35]: Can you pull the specific rename notes from that pull request?

File: RELEASE-NOTES.md (hunk 1 above)
Comment (madkinsz) [inline RELEASE-NOTES.md:31]:
```suggestion
- Fix bug where events worker would fail if the API URL including a trailing `/` — https://github.com/PrefectHQ/prefect/pull/9663
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,60 @@
...
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix inconsistent flow run parameters when creating a new flow run or editing a deployment — https://github.com/PrefectHQ/prefect/pull/9705
+- Fix max value when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
+- Allow table artifacts `table` argument as list of lists — https://github.com/PrefectHQ/prefect/pull/9732
+- Use `PrefectHttpxClient` in cloud communication — https://github.com/PrefectHQ/prefect/pull/9724
```
Comment (madkinsz) [inline RELEASE-NOTES.md:30]: We can move to enhancements — would rephrase as "Use base client for Prefect Cloud communication to include automatic retries"

Comment (madkinsz) [reply RELEASE-NOTES.md:30]: or "Add request retry support to Prefect Cloud client"

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,60 @@
...
+
+### Enhancements
+- Emit events from deployments, work queues, and work pools. — https://github.com/PrefectHQ/prefect/pull/9635
+- Improve SQLite database transaction behavior — https://github.com/PrefectHQ/prefect/pull/9594
+- Add support for SQLAlchemy 2 — https://github.com/PrefectHQ/prefect/pull/9656
+- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
+- Add an `/api/ready check` endpoint to Prefect server — https://github.com/PrefectHQ/prefect/pull/9701
+- Guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Handle deleted deployments gracefully — https://github.com/PrefectHQ/prefect/pull/9464
```
Comment (madkinsz) [inline RELEASE-NOTES.md:18]: "Fix bug where agents would fail when processing runs with deleted deployments"

more out of enhancements

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,60 @@
...
+- Add interactive prompts for deployment name and work pool to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/9707
+- Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
+
+### Enhancements
+- Emit events from deployments, work queues, and work pools. — https://github.com/PrefectHQ/prefect/pull/9635
+- Improve SQLite database transaction behavior — https://github.com/PrefectHQ/prefect/pull/9594
+- Add support for SQLAlchemy 2 — https://github.com/PrefectHQ/prefect/pull/9656
+- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
+- Add an `/api/ready check` endpoint to Prefect server — https://github.com/PrefectHQ/prefect/pull/9701
+- Guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
```
Comment (madkinsz) [inline RELEASE-NOTES.md:16]:
```suggestion
- Add guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,65 @@
...
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix inconsistent flow run parameters when creating a new flow run or editing a deployment — https://github.com/PrefectHQ/prefect/pull/9705
+- Fix max value when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
+- Allow table artifacts `table` argument as list of lists — https://github.com/PrefectHQ/prefect/pull/9732
+- Use `PrefectHttpxClient` in cloud communication — https://github.com/PrefectHQ/prefect/pull/9724
+- Remove trailing slashes when building events/in websocket url — https://github.com/PrefectHQ/prefect/pull/9663
+- Update flow run logs empty text for light mode
```
Comment (pleek91) [inline RELEASE-NOTES.md:32]:
```suggestion
- Unknown states crash flow run timeline — https://github.com/PrefectHQ/prefect-ui-library/pull/1448
```

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,60 @@
...
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Handle deleted deployments gracefully — https://github.com/PrefectHQ/prefect/pull/9464
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix inconsistent flow run parameters when creating a new flow run or editing a deployment — https://github.com/PrefectHQ/prefect/pull/9705
+- Fix max value when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
```
Comment (madkinsz) [inline RELEASE-NOTES.md:28]: Max value of what?

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,60 @@
...
+- Guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Handle deleted deployments gracefully — https://github.com/PrefectHQ/prefect/pull/9464
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix inconsistent flow run parameters when creating a new flow run or editing a deployment — https://github.com/PrefectHQ/prefect/pull/9705
```
Comment (madkinsz) [inline RELEASE-NOTES.md:27]: I think this is UI specific

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,60 @@
...
+- Add an `/api/ready check` endpoint to Prefect server — https://github.com/PrefectHQ/prefect/pull/9701
+- Guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Handle deleted deployments gracefully — https://github.com/PrefectHQ/prefect/pull/9464
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Add worker event telemetry — https://github.com/PrefectHQ/prefect/pull/9702
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables exist in base scope - https://github.com/PrefectHQ/prefect/pull/9665
```
Comment (madkinsz) [inline RELEASE-NOTES.md:26]:
```suggestion
- Fix hanging flow runs from deployments when variables retrieved in base scope - https://github.com/PrefectHQ/prefect/pull/9665
```

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -1,5 +1,68 @@
...
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables retrieved in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix maximum character length when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
+- Fix bug where agents would fail when processing runs with deleted deployments — https://github.com/PrefectHQ/prefect/pull/9464
+- Fix bug where `uvicorn` could not be found when server was started from an unloaded virtual environment - https://github.com/PrefectHQ/prefect/pull/9734 
+- Allow table artifacts `table` argument as list of lists — https://github.com/PrefectHQ/prefect/pull/9732
+- Add request retry support to Prefect Cloud client — https://github.com/PrefectHQ/prefect/pull/9724
+- Fix bug where events worker would fail if the API URL includes a trailing `/` — https://github.com/PrefectHQ/prefect/pull/9663
+- Unknown states crash flow run timeline — https://github.com/PrefectHQ/prefect-ui-library/pull/1448
```
Comment (madkinsz) [inline RELEASE-NOTES.md:34]: ?

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -1,5 +1,68 @@
...
+- Update docs links in ui to concepts when possible — https://github.com/PrefectHQ/prefect-ui-library/pull/1351
+
+### Fixes
+- Display CLI deprecation warnings to STDERR instead of STDOUT — https://github.com/PrefectHQ/prefect/pull/9690
+- Fix hanging flow runs from deployments when variables retrieved in base scope - https://github.com/PrefectHQ/prefect/pull/9665
+- Fix maximum character length when updating variables — https://github.com/PrefectHQ/prefect/pull/9710
+- Fix bug where agents would fail when processing runs with deleted deployments — https://github.com/PrefectHQ/prefect/pull/9464
+- Fix bug where `uvicorn` could not be found when server was started from an unloaded virtual environment - https://github.com/PrefectHQ/prefect/pull/9734 
+- Allow table artifacts `table` argument as list of lists — https://github.com/PrefectHQ/prefect/pull/9732
+- Add request retry support to Prefect Cloud client — https://github.com/PrefectHQ/prefect/pull/9724
```
Comment (madkinsz) [inline RELEASE-NOTES.md:32]: This is an enhancement

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -1,5 +1,68 @@
...
+- Add `on_cancellation` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9389
+- Improve cancellation cleanup service iteration over subflow runs - https://github.com/PrefectHQ/prefect/pull/9731
+- Add `PREFECT_CLIENT_MAX_RETRIES` for configuration of maximum HTTP request retries - https://github.com/PrefectHQ/prefect/pull/9735 
+- Add an `/api/ready` endpoint to the Prefect server to check database connectivity — https://github.com/PrefectHQ/prefect/pull/9701
+- Display URL to flow run on creation - https://github.com/PrefectHQ/prefect/pull/9740 
+- Add guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Emit worker event when a flow run is scheduled to run or cancel — https://github.com/PrefectHQ/prefect/pull/9702
+- Add "Retrying" to ui states select — https://github.com/PrefectHQ/prefect-ui-library/pull/1410
```
Comment (madkinsz) [inline RELEASE-NOTES.md:22]: ? What does this mean to a user?

File: RELEASE-NOTES.md [hunk 17]
```diff
@@ -1,5 +1,68 @@
...
+- Update docs links in ui to concepts when possible — https://github.com/PrefectHQ/prefect-ui-library/pull/1351
```
Comment (madkinsz) [inline RELEASE-NOTES.md:23]: Does this belong in the docs section?
```suggestion
- Update docs links in UI to concepts when possible — https://github.com/PrefectHQ/prefect-ui-library/pull/1351
```

Comment (serinamarie) [reply RELEASE-NOTES.md:24]: I think that makes sense

File: RELEASE-NOTES.md (hunk 14 above)
Comment (serinamarie) [reply RELEASE-NOTES.md]: Yes 👍

File: RELEASE-NOTES.md [hunk 18]
```diff
@@ -1,5 +1,67 @@
 # Prefect Release Notes
 
+## Release 2.10.11
+
+- Adds work pool creation wizard — https://github.com/PrefectHQ/prefect/pull/9717
+- Add interactive prompts for deployment name and work pool to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/9707
+- Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
+
+### Enhancements
+- Emit events from deployments, work queues, and work pools. — https://github.com/PrefectHQ/prefect/pull/9635
```
Comment (discdiver) [inline RELEASE-NOTES.md:10]:
```suggestion
- Emit events from deployments, work queues, and work pools — https://github.com/PrefectHQ/prefect/pull/9635
```

**Review by discdiver — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 19]
```diff
@@ -1,5 +1,82 @@
...
+- Improve cancellation cleanup service iteration over subflow runs - https://github.com/PrefectHQ/prefect/pull/9731
+- Add request retry support to Prefect Cloud client — https://github.com/PrefectHQ/prefect/pull/9724
+- Add `PREFECT_CLIENT_MAX_RETRIES` for configuration of maximum HTTP request retries - https://github.com/PrefectHQ/prefect/pull/9735 
+- Add an `/api/ready` endpoint to the Prefect server to check database connectivity — https://github.com/PrefectHQ/prefect/pull/9701
+- Display URL to flow run on creation - https://github.com/PrefectHQ/prefect/pull/9740 
+- Add guard against changing the profile path from `prefect config set` — https://github.com/PrefectHQ/prefect/pull/9696
+- Use flow run logger to report traceback for failed submissions — https://github.com/PrefectHQ/prefect/pull/9733
+- Improve default Prefect image tag when using development versions — https://github.com/PrefectHQ/prefect/pull/9503
+- Emit worker event when a flow run is scheduled to run or cancel — https://github.com/PrefectHQ/prefect/pull/9702
+- Add ability in the Prefect UI to filter Task Runs tab for `Retrying` state — https://github.com/PrefectHQ/prefect-ui-library/pull/1410
```
Comment (billpalombi) [inline RELEASE-NOTES.md:38]:
```suggestion
- Add ability to filter for `Retrying` state in the Task Runs tab of the Prefect UI — https://github.com/PrefectHQ/prefect-ui-library/pull/1410
```

**Review by billpalombi — APPROVED** [review]
Oh nit, otherwise lgtm

**Review by madkinsz — APPROVED** [review]
